Dockerfile

```diff
@@ -99,34 +99,38 @@ RUN gem install \
 RUN rm -rf /usr/local/bundle/cache/* \
  && find /usr/local/bundle/ -name "*.o" | xargs rm
 
-FROM fluent/fluentd:v1.14.4-debian${FLUENTD_ARCH}-1.0
+FROM fluent/fluentd:v1.14.5-debian${FLUENTD_ARCH}-1.0
```

Comment:
fluentd version should be changed for all plugins as well

Reply:
also here https://github.com/SumoLogic/sumologic-kubernetes-fluentd/blob/137628ce1faa4a3897b4e22731757135238dd8b6/Dockerfile#L26 should be `1.14.5`